modules/core/src/test/scala/compiler/Environment.scala

```diff
@@ -1,10 +1,11 @@
-// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
 package compiler
 
 import cats.Id
 import cats.implicits._
+import cats.catsInstancesForId
```

Comment:
Scala 3 can't find instances for `cats.Id` but it's not clear why. This fixes it.